ResourceRest/PayloadAliasRegistry.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ResourceRestBundle\ResourceRest;
```

Comment:
Well, I do this with the aim of creating a component .. but in this case we could move it to the bundle root.